8.0/debian/buster/developer/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM php:8.0-buster
+MAINTAINER "Bence Gazder"
+LABEL maintainer="bence.gazder@apicore.de"
+
+ENV COMPOSER_VERSION 2.0.11
+
+# install dependencies
+RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini" && \ 
+    apt-get update && apt-get install -y \
+    supervisor \
+    cron \
+    ca-certificates \
+    libfreetype6 \
+    libfreetype6-dev \
+    libssl1.0 \
+    libpng16-16 \
+    libjpeg62-turbo \
+    libjpeg62-turbo-dev \
+    mcrypt \
+    libcurl4-openssl-dev \
+    libpng-dev \
+    libicu-dev \
+    libonig-dev \
+    zip \
+    libzip-dev \
+    $PHPIZE_DEPS \
+    nano \
+    git \
+    openssh-client \
+    sudo \
```

Comment:
The sudo package should be required in an image like this. The images are usually single-user (from the runner's point of view) so it either has a permission or not. If the entrypoint drops privileges, it shouldn't want to elevate again.

Please remove from every image.

Reply:
If I remove this, it will break already present `run.sh` scripts. First we need to refactor startup scripts

Reply:
If you remove it only in the 8.0 version, it will be just a standard procedure for upgrading to the - most probably - already breaking change that is php 8.0

Reply:
Okay I'll remove it from the 8.0 version

Reply:
Removed from 8.0 images